.npmrc

```diff
@@ -0,0 +1,2 @@
+registry=http://npm.bbpd.io/repository/bb-npm/
```

Comment:
Do we need this private registry since we already publish this in Github?

Reply:
Good point. We should remove it.